woocommerce/woocommerce-android
# Save Order list/Order detail navigation state when the fragment recreates

Closes: #10576 


### Description

This PR addresses the management of Order list/Order detail navigation state during configuration changes and dynamically adjusts to variations in app window size, such as when entering split view or resizing the window. It ensures consistent navigation behaviour across these scenarios.

### Testing instructions

Ensure the Order list/ Order detail screen states are maintained in the following scenarios
1. On configuration change such as screen rotation
2. When the app enters into split view and the app window is resized

**Note:** There's a known issue for which I've created a ticket here: https://github.com/woocommerce/woocommerce-android/issues/10700

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/1331230/241e88a4-7445-4089-b1ac-d1906eafa7be
